src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/Confirm.cshtml

```diff
@@ -72,24 +76,25 @@
     </div>
 </div>
 
-
-<ol class="govuk-list app-status-list">
-    <li confirmation-section asp-page="ConfirmYourEmployer" model="Model">
-        Your employer
-    </li>
-    <li confirmation-section asp-page="ConfirmYourTrainingProvider" model="Model">
-        Your training provider
-    </li>
-    <li confirmation-section asp-page="ConfirmYourApprenticeshipDetails" model="Model">
-        Your apprenticeship details
-    </li>
-    <li confirmation-section asp-page="HowYourApprenticeshipWillBeDelivered" model="Model">
-        How your apprenticeship will be delivered
-    </li>
-    <li confirmation-section asp-page="RolesAndResponsibilities" model="Model">
-        Roles and responsibilities
-    </li>
-</ol>
+<div asp-show="@Model.Status != ConfirmStatus.Stopped">
```

Comment:
Is adding logic like this make our implementation more complex than it needs to be. 

I think you've implemented stopped view very well and I'm happy with how it works, but it does feel like a classic ASP script where we start to build the page dynamically according to the status. 

Is it worthwhile implementing the Controller Pattern (instead of the Page Pattern we currently use) and rendering a Stopped view and the confirmation view as 2 distinct views? 

Ben may have an opinion on this, so I'm happy to have this comment ignored if you are both happy with this pattern

Reply:
There are several sections on the page shown depending on the ConfirmStatus

Reply:
I haven't had to write a page that has completely different content based on state in Razor Pages before - I don't know what the best technique is.

If this is something we do again I'd like to spend some time understanding the options and see what works best.

For now I'm happy to have it like this.  I like using the `<partial>` for Stopped - that prevents the page from being overwhelmed with the `if/else` condition.